src/shared/components/SubmissionManagement/SubmissionManagement/index.jsx

```diff
@@ -72,10 +72,17 @@ export default function SubmissionManagement(props) {
         </div>
         <div styleName="right-col">
           <p styleName="round">{currentPhase.phaseType}</p>
-          <p styleName="time-left">
-            {days > 0 && (`${days}D`)} {hours}H {minutes}M
-          </p>
-          <p styleName="left-label">left</p>
+          {challenge.status !== 'COMPLETED' &&
```

Comment:
For cleaner code, please combine these two conditional blocks into one like:
```
{
  challenge.status === 'COMPLETED' ? (
    /* Markup for this case */
  ) : (
    /* Render markup for this case */
  )
}
```
Otherwise the code looks correct

Reply:
@birdofpreyru updated